charts/redpanda/values.yaml

```diff
@@ -219,6 +219,8 @@ tls:
       # that are not authenticated using public authorities.
       caEnabled: true
       # duration: 43800h
+      # if you wish to for apply internal dns names to the certificate even when supplying an issuer
+      # applyInternalDNSNames: false
```

Comment:
Any reason to not include the actual value here instead of commenting it out?

We'll want the type of the values struct to match the type of `values.yaml`. If we don't include the value in `values.yaml`, then `ApplyInternalDNSNames` should be a pointer to a bool (`*bool`) to indicate that `nil` is an acceptable value.

Reply:
yes its a pointer, there is no need for this to be included in the api, it did not exist before and we are using dig so we can ignore it if missing.

Reply:
I don't think I understand your reasoning for not including this with a default value of `false`.

As the field exists on our values struct and therefore JSONSchema, it's part of the API. Including the default in values.yaml is a backward compatible change as the users' values will be merged with it. Including the default also makes it much easier to reason about the the behavior because there are two states (true / false) vs three states (true / false / nil).